runner_option.go

```diff
@@ -20,11 +23,16 @@ type httpRunnerConfig struct {
 	SkipVerify           bool   `yaml:"skipVerify,omitempty"`
 	Timeout              string `yaml:"timeout,omitempty"`
 	UseCookie            *bool  `yaml:"useCookie,omitempty"`
-	Trace                *bool  `yaml:"trace,omitempty"`
+	Trace                traceConfig
 
 	openApi3Doc *openapi3.T
 }
 
+type traceConfig struct {
+	Enable     *bool  `yaml:"enable,omitempty"`
+	Headername string `yaml:"headername,omitempty"`
```

Comment:
Is there any reason why it shouldn't be `headerName`?

```suggestion
	HeaderName string `yaml:"headerName,omitempty"`
```